pages/conduct.html

```diff
@@ -37,9 +38,9 @@
 <ol>
   <li>
     <p>
-      Harassment includes offensive verbal comments related to gender,
+      Harassment includes offensive verbal or written comments related to gender,
       sexual orientation, disability, physical appearance, body size,
-      race, religion, sexual images in public spaces, deliberate
+      race, or religion, sexual images in public spaces, deliberate
```

Comment:
This “or” is a step in the right direction.  But there's a lot of duplication between this paragraph and the previous paraagraph.  I'd suggest a more thorough rewrite to make the previous paragraph the source of ok things for which you should not be harassed (e.g. it's ok to have a religion).  Then this paragraph could be not-ok things that count as harassment (e.g. posting sexual images).